lib/Runtime/Library/RegexHelper.cpp

```diff
@@ -1213,7 +1212,156 @@ namespace Js
     }
 
     // String.prototype.split (ES5 15.5.4.14)
-    Var RegexHelper::RegexSplitImpl(ScriptContext* scriptContext, JavascriptRegExp* regularExpression, JavascriptString* input, CharCount limit, bool noResult, void *const stackAllocationPointer)
+    Var RegexHelper::RegexSplitImpl(ScriptContext* scriptContext, RecyclableObject* thisObj, JavascriptString* input, CharCount limit, bool noResult, void *const stackAllocationPointer)
```

Comment:
I created a new function called `RegexEs5SplitImpl` for the ES5 version, but looks like I forgot to move the spec reference :smile: